server/service/integration_enterprise_test.go

```diff
@@ -10707,13 +10708,47 @@ func (s *integrationEnterpriseTestSuite) TestSoftwareInstallerUploadDownloadAndD
 		// upload again fails
 		s.uploadSoftwareInstaller(t, payload, http.StatusConflict, "already exists")
 
+		// patch the software installer to change the labels
+		var b bytes.Buffer
+		w := multipart.NewWriter(&b)
+		require.NoError(t, w.WriteField("team_id", "0"))
+		require.NoError(t, w.WriteField("labels_exclude_any", t.Name()))
+		w.Close()
+		headers := map[string]string{
+			"Content-Type":  w.FormDataContentType(),
+			"Accept":        "application/json",
+			"Authorization": fmt.Sprintf("Bearer %s", s.token),
+		}
```

Comment:
No need to change, but FYI you can now use `generateMultipartRequest` without a file to upload (it will skip the file if the filename+content are empty). I updated it in the activities PR for that purpose.